python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -20,12 +41,17 @@
 from dagster._core.workspace.workspace import IWorkspace
 from dagster._daemon.daemon import DaemonIterator, IntervalDaemon
 from dagster._utils import hash_collection
+from dagster._utils.error import serializable_error_info_from_exc_info
 
 CURSOR_KEY = "ASSET_DAEMON_CURSOR"
 ASSET_DAEMON_PAUSED_KEY = "ASSET_DAEMON_PAUSED"
 
 EVALUATIONS_TTL_DAYS = 30
 
+FIXED_AUTO_MATERIALIZATION_ORIGIN_ID = "amp"
```

Comment:
maybe these should be renamed to something along the lines of `FIXED_ASSET_DAEMON_ORIGIN_ID = "assetdaemon"`, as this is a bit more general.